src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -109,7 +111,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     OutDir can be used to gather multiple project outputs in one location. In addition,
     OutDir is included in AssemblySearchPaths used for resolving references.
 
+    BaseOutputPath:
+    This is the top level folder where all configuration specific output folders will be created.
+    Default value is bin\
+
     OutputPath:
+    This is the full Output Path, and is derived from BaseOutputPath, if none specified
+    (eg. bin\Debug). If this property is overridden, then setting BaseOutputPath has no effect.
+
+    For projects loaded with MPFProj (Legacy project system):
```

Comment:
What is MPFProj?

Reply:
Isn't that a common name for legacy project system for `.NET` (`csproj.dll` and `msvbprj.dll`)!?

If not, then, what's the correct term?